docs-chef-io/content/inspec/troubleshooting.md

```diff
@@ -22,3 +22,41 @@ See the relevant resource pack readme for instructions:
 - [inspec-aws README](https://github.com/inspec/inspec-aws#use-the-resources)
 - [inspec-azure README](https://github.com/inspec/inspec-azure#use-the-resources)
 - [inspec-gcp README](https://github.com/inspec/inspec-gcp#use-the-resources)
+
+## License is not entitled to use InSpec
+
+The error indicates the license key used with Chef InSpec, version 6 or higher, is not entitled to the software Chef InSpec. To resolve this issue you need to use a license key which is entitled to Chef InSpec.
+<!-- TODO Need to check the validity of below license information link-->
+See the [Chef License documentation](/license/) for more information about Chef licensing.
+
+## Unable to connect to the licensing server. InSpec requires server communication to operate
+
+The error indicates that Chef InSpec, version 6 or higher, is having trouble connecting to the Licensing Service. This service is responsible for validating the Chef license key used with Chef InSpec. The Licensing Service can either be a Chef's Licensing Service or it can be configured to point to Local Licensing Service.
+
+Here are a few steps to troubleshoot and resolve this issue:
+
+- Check Network Connectivity: Ensure that the machine running Chef InSpec has proper network connectivity. It should be able to reach the Licensing Service's endpoint. This includes checking firewall settings and network configuration.
+
+- Service Availability: If it's a Local Licensing Service, ensure that it's properly configured and operational. Any misconfigurations or issues with the local service could lead to connection problems.
+
+- URL Configuration: Verify that the URL configured in Chef InSpec is correct. If the URL is configured using an environment variable, check the value for environment variable `CHEF_LICENSE_SERVER` to confirm the configured URL.
```

Comment:
Would the `inspec license list` command help with this?

Reply:
If someone uses the env variable, is that stored in the InSpec config file?

Reply:
The value is set only in env and not in inspec config. But we store this value within `licenses.yaml`